### PR TITLE
Add tests for TraceShim/TracerShim.

### PR DESCRIPTION
**Pull request by carlosalberto**
Although we have the extensive testbed section,
we need some basic tests for non-exotic scenarios,
and this is part of that intention.

### CODE REVIEW

File: opentracing_shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java [hunk 1]
```diff
@@ -16,27 +16,69 @@
...
     SpanContext result =
-        testClass.extract(
+        tracerShim.extract(
             Format.Builtin.TEXT_MAP, new TextMapAdapter(Collections.<String, String>emptyMap()));
     assertNull(result);
   }
+
+  @Test
+  public void inject_nullContext() {
+    tracerShim.inject(
```
Comment (pavolloffay) [inline opentracing_shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java]: nit: assert on empty map

Comment (bogdandrutu) [reply opentracing_shim/src/test/java/io/opentelemetry/opentracingshim/TracerShimTest.java]: @carlosalberto please fix.

**Comment by carlosalberto** [comment]
Hey @pavolloffay Updated the bit (confirm this is what you wanted ;) )

I will either merge or update, depending on what you comment.

**Comment by carlosalberto** [comment]
Merged. Let me know if you think I missed something (and maybe open an Issue with it? ;) )

**Comment by pavolloffay** [comment]
> Hey @pavolloffay Updated the bit (confirm this is what you wanted ;) )

It's perfect thanks!

**Pull request merged**